src/components/main/Main.js

```diff
@@ -30,15 +30,15 @@ class Main extends Component {
   }
 
   render() {
-    const {title} = this.props;
-    const {users, posts, date} = this.state;
+    const { title } = this.props;
+    const { users, posts, date } = this.state;
 
     return (
       <main className="main" title={title}>
         <Aside/>
         <Content/>
-        <UserList items={users} handleClick={(e) => this.showUserName(e)}/>
-        <Posts posts={posts}/>
+        <UserList items={users} handleClick={(e) => this.getUserPosts(e)}/>
+        {(posts.length !== 0) && <Posts posts={posts}/>}
```

Comment:
no need round brackets, also you can just check `posts.length && <Posts posts={posts} />`